### DISCUSSION TITLE
dde-dock 的 plugins developer guide 老旧

### DISCUSSION THREAD

**Discussion by tensor-tech** [linuxdeepin/developer-center#7477] [Development Questions | 开发答疑]
dde-dock 的 plugins developer guide 太老旧了，希望能更新一下，里面有很多问题：
1.  QIcon icon(const DockPart &dockPart, int themeType) override;  这个函数已经改成  QIcon icon(const DockPart &dockPart, DGuiApplicationHelper::ColorType th
2. PluginFlags 改为 PluginFlag

**Comment by BLumia** [comment]
23 用的 dock 会有其它的变化，*可能* 整个文档会过时。正式版后再更新整体的 dock 文档吧。

(ps. 你提到的问题，如果你愿意的话可以提个 PR 修正，我们帮你合进去。至少可以帮助到这段时间内查阅文档编写插件的人）

cc @tsic404
